website/content/docs/ui/custom-messages.mdx

```diff
@@ -0,0 +1,151 @@
+
+1. On the **Create message** form:
+
+   - Select the locations where your message should appear.
+   - Select the message type.
+   - Provide a title and the message text. For important messages, we recommend
+     keeping the text short and including a link to more information rather
+     than writing a longer message that users may not take the time to read.
+   - Set a start time when your message will publish. By default, messages
+     publish at midnight in your local timezone.
+   - Set an optional end time for the message. By default, messages do not
+     expire.
+
+1. Click **Preview** to see how your message will appear to users.
+
+1. Click **Create message** to save your new message.
+
+## Create messages for a specific namespace
+
+Child [namespaces](/vault/docs/enterprise/namespaces) inherit all messages
+created on the parent namespace. For example, if your cluster has the following
+namespace hierarchy:
+
+<CodeBlockConfig hideClipboard>
+
+```plaintext
+─ admin
+    ├── finance
+    └── marketing
+        ├── digital-marketing
+        └── events
+```
+
+</CodeBlockConfig>
+
+Custom messages created on the `admin` namespace apply to all child namespaces:
+`fiance`, `marketing`, `marketing/digital-marketing` and `marketing/events`.
+
+To create a custom message that only targets the marketing team you,
+must log into the `marketing` namespace before creating your message.
+
+To create a message under a specific namespace:
+
+1. On your Vault login page, enter the namespace you want to target in
+   the **Namespace** text field.
+1. Select an appropriate authentication method and log in.
+
+1. Select **Custom Messages**.
+
+1. Select the **+ Create message** button in the toolbar to open and fill out
+   the **Create message** form.
+1. Click **Preview** to see how your message will appear to users.
+
+1. Click **Create message** to save your new message.
+
+Your new message only appears when a user logs into the targeted
+namespace or one of its child namespaces. You can verify that your
+message has the correct behavior by logging into an admin or parent
+namespace. The message should only appear when you switch to the
+targeted namespace.
+
+![namespace picker to change namespaces](/img/ui-custom-msg.png)
+
+## Edit a custom message
+
+You can open the edit screen for a custom message in two places from the **Custom messages** page.
+
+**From the additional options menu for the message**:
+
+1. Find the custom message you want to edit.
+1. Click the additional option button (three dots).
+1. Select **Edit** from the dropdown menu to bring up the edit page.
+
+**From the message details page**:
+
+1. Find the custom message you want to edit.
+1. Select the message to open the message details page.
+1. Click the **Edit message** button to bring up the edit page.
+   After clicking **Edit message** in either of the options, fill in the information that you would like to edit for the custom message.
+
```

Comment:
this line is confusing -- I suggest removing "either of the options" so this flows more smoothly